CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### 🚀 Added
+- Add fixer: LeadingCharacter [#255](https://github.com/dotenv-linter/dotenv-linter/pull/259) ([@baile320](https://github.com/baile320))
```

Comment:
```suggestion
- Add fixer: LeadingCharacter [#259](https://github.com/dotenv-linter/dotenv-linter/pull/259) ([@baile320](https://github.com/baile320))
```

Reply:
Sorry, typo! Fixed in this commit: https://github.com/dotenv-linter/dotenv-linter/pull/259/commits/b2b50c76a30ff494b62119aed48bacf159236c04